x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/Authentication.java

```diff
@@ -72,10 +72,6 @@ public class Authentication implements ToXContentObject {
     private final AuthenticationType type;
     private final Map<String, Object> metadata; // authentication contains metadata, includes api_key details (including api_key metadata)
 
-    public Authentication(User user, RealmRef authenticatedBy, RealmRef lookedUpBy) {
-        this(user, authenticatedBy, lookedUpBy, Version.CURRENT, AuthenticationType.REALM, Collections.emptyMap());
-    }
```

Comment:
This constructor is now removed. The next PR will make the other constructor private.

Reply:
After a second thought, I restored this constructor for this PR because it is tagged as `>test` and this is technically production code. I will raise a separate PR _after_ all test usages are addressed to remove this constructor and change the other to private.